### DISCUSSION TITLE
Why do you use FFT and other images transformation?

### DISCUSSION THREAD

**Discussion by tempdeltavalue** [eps696/aphantasia#37] [General]
I saw same thing here,

https://github.com/greentfrapp/lucent
and
https://github.com/Mayukhdeb/torch-dreams/commits?author=Mayukhdeb

And I'm not really sure what I understand that...

Sorry to bother you, just curious 🙌

**Comment by eps696** [comment]
i'm afraid, i didn't quite understand your question. 
confirmed [that's written in the very beginning of readme] - this generation method is based on the fft/ifft transforms, taken from the Lucent library (torch dreams by Mayukhdeb appeared later). why? just because ::]

on a seirous note - other CLIP-based text-to-image methods usually employed fat pretrained GAN generators, while i believed comparable [or better] results could be achieved with such lightweight ops as fft/dwt - if only i understood it enough to use and configure properly..

**Comment by tempdeltavalue** [comment]
My question was why you need fft for image generation at all ... but you already answered :)

**Reply by tempdeltavalue** [reply]
Probably interesting video to check in current context

https://www.youtube.com/watch?v=Al6NTbgka1o

**Comment by eps696** [comment]
yeah, good video, thanks. 
the method itself has been published by another Matthew [2 years ago](https://github.com/tancik/fourier-feature-networks) 
[and also used in the very first tool from this repo - [SIREN + FF mapping](https://colab.research.google.com/drive/1L14q4To5rMK8q2E6whOibQBnPnVbRJ_7)]
